### PR TITLE
Add PDF flavor to ScanStrings

### PR DESCRIPTION
**Pull request by jkamdjou**
_This is all @morriscode's handywork, I take no credit! 😄 We're just going through the paces of making PRs together._

**Describe the change**
Added PDF files to Strings and Exiftools scanner.

**Describe testing procedures**
Describe the tests that you ran to verify your changes (including test configurations) and instructions so we can reproduce the tests. To assist in testing, the project maintainers may ask for file samples.

Note: still need to test.

**Sample output**
If this change modifies Strelka's output, then please include a sample of the output here.

Sample of strings output on PDF File:
```
pdf:{} 1 item
strings:{} 1 item
strings:[] 5422 items
0:%PDF-1.4
1:1 0 obj
2:<</Type/XObject/Subtype/Image/Width 842/Height 572/Length 490/ColorSpace/DeviceGray/BitsPerComponent 8/Filter/FlateDecode>>stream
3:endstream
4:endobj
5:2 0 obj
6:<</Type/XObject/Subtype/Image/Width 842/Height 572/SMask 1 0 R/Length 415661/ColorSpace[/CalRGB<</Gamma[2.2 2.2 2.2]/Matrix[0.41239 0.21264 0.01933 0.35758 0.71517 0.11919 0.18045 0.07218 0.9504]/WhitePoint[0.95043 1 1.09]>>]/BitsPerComponent 8/Filter/FlateDecode>>stream
```

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of and tested my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: build/configs/scanners.yaml [hunk 1]
```diff
@@ -93,6 +93,7 @@ scanners:
           - 'vbframe_file'
           - 'mso_file'
           - 'application/vnd.ms-excel'
+          - 'pdf_file'
```
Comment (cameron-dunn-sublime) [inline build/configs/scanners.yaml:96]: `pdf_file` is positive for `ScanExiftool` down on line 120. A different set of options are selected though.

Comment (morriscode) [reply build/configs/scanners.yaml:96]: Nice catch guess i didn't scroll far enough :)

Comment (morriscode) [reply build/configs/scanners.yaml]: updated

**Comment by morriscode** [comment]
@cameron-dunn-sublime  just a heads up! 🙏

**Review by cameron-dunn-sublime — APPROVED** [review]

**Pull request merged**